adapters/adhese/adhese.go

```diff
@@ -124,6 +124,7 @@ func (a *AdheseAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adap
 	return []*adapters.RequestData{{
 		Method: "GET",
 		Uri:    complete_url,
+		ImpIDs: openrtb_ext.GetImpIDs(request.Imp),
```

Comment:
Looks like this should be just the id for imp[0]. There is no body and the url is composed only of the first impression, thus ignoring all impressions except the first.

Reply:
Changed to single imp.Id